scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/ItemsListGrid.java

```diff
@@ -230,14 +231,10 @@ protected void applyCurrentLocalFilter() {
         request.setEndRow(nbOfItems + visibleRows[1]);
         request.setSortBy(this.getSort());
 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)